### DISCUSSION TITLE
Is the time provided by Stellarium Web UTC time?

### DISCUSSION THREAD

**Discussion by Runmartinrun** [Stellarium/stellarium#1710] [Q&A]
Hello,
I am using [stellarium web](https://stellarium-web.org/), and I would like to know what is the exact time format of the time reported in the bottom right side of the screen. Is this the UTC time corresponding to the location of the observer on the date used? 

For example, if the location is Paris and the date is June 1 2021, is that time equal to UTC+02 time?

Thanks!

**Comment by guillaumechereau** [comment]
Yes it is UTC, in the timezone of your local location.
